frontend/src/metabase/dashboard/components/DashboardGrid.jsx

```diff
@@ -294,6 +295,7 @@ class DashboardGrid extends Component {
   };
 
   onDashCardRemove(dc) {
+    this.props.cancelFetchCardData(dc.card_id, dc.id);
```

Comment:
I'd put this call directly into `removeCardFromDashboard`. You'd need make it a `ThunkAction`. Search for actions in `data-fetching.js` for examples

Reply:
Moved the call in the action/thunk, thank you for the suggestion